build/manifest/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/mattermost/mattermost-plugin-demo/build/manifest
 require (
 	github.com/gorilla/websocket v1.2.0 // indirect
 	github.com/hashicorp/go-hclog v0.0.0-20180402200405-69ff559dc25f // indirect
-	github.com/mattermost/mattermost-server v0.0.0-20180718195328-2edc5d4fe051
+	github.com/mattermost/mattermost-server v5.1.0-rc1.0.20180727213543-6ac82d517176+incompatible
```

Comment:
Can you share how you updated `go.mod`? It's odd that it would point at `5.1.0` instead of `master` with the actual latest manifest changes. Anyway, I have a set of changes to rip out `vgo` as discussed anyway, but more curious than anything.

Reply:
I just typed in `master` vgo changed it to that.